### ISSUE TITLE
Support for Deno packages

### ISSUE THREAD

**Issue by amitav13** [vercel/turbo#6368]
### Which project is this feature idea for?

Turborepo

### Describe the feature you'd like to request

We currently have supabase edge functions (written in TS) sitting in a package that we deploy independently of the rest of our Node/React packages and apps. We also have a `common` package that houses types and other utils that we use across our apps. We want to pull these in to our supabase functions as well.

### Describe the solution you'd like

I'd ideally want to just import the `common` package like we do in the rest of our apps, a line in package.json - 

`"common": "workspace:*"`

### Describe alternatives you've considered

A workaround would be to publish our package on npm and then add it in as a regular `esm.sh` import from a URL - though we lose all the goodness from having it as an internal dependency.

**Comment by tknickman** [comment]
Hey @amitav13, thanks for the issue. This is something that isn't immediately on the roadmap. Deno is pretty different in that there is no package manager or lockfile - everything is url based. Even the common standard of storing and exporting dependencies from `deps.ts` and `dev_deps.ts` is just a convention, and not enforced. This would make integrating turborepo difficult as the dependency paradigm is quite different. 

I'll leave this issue open so we can continue to gauge interest from the community, but as of now it's not something we're planning on in the short term.

**Comment by amitav13** [comment]
> Hey @amitav13, thanks for the issue. This is something that isn't immediately on the roadmap. Deno is pretty different in that there is no package manager or lockfile - everything is url based. Even the common standard of storing and exporting dependencies from `deps.ts` and `dev_deps.ts` is just a convention, and not enforced. This would make integrating turborepo difficult as the dependency paradigm is quite different.
> 
> I'll leave this issue open so we can continue to gauge interest from the community, but as of now it's not something we're planning on in the short term.

Yeah I get that, its a hard one to pull in. I don't believe Nx or any other monorepo tool is supporting this atm. Would still be great to have though, we're just copy/pasting with a loud comment to remember to update in 2 places 😅
